README.md

```diff
@@ -35,7 +35,23 @@ To create the necessary infrastructure run the script ```./create-ionos-kubernet
 ./create-ionos-kubernetes.sh
 ```
 
-### 2. cert-manager (optional)
+### 2. nginx-ingress (optional)
+
+```bash
+export KUBECONFIG=$(readlink -f kubeconfig-ionos.yaml)
+
+helm repo add nginx-stable https://helm.nginx.com/stable
+helm repo update
+# Install nginx-ingress
+helm install \
+  nginx-ingress nginx-stable/nginx-ingress \
+  --namespace nginx-ingress \
+  --create-namespace \
+  --version v0.16.2 \
+  --create-namespace
+```
+
+### 3. cert-manager (optional)
```

Comment:
NGINX-Ingress chart is already installed via terraform and we do not need to install it again!